pac4cli.service

```diff
@@ -3,7 +3,7 @@ Description=PAC autoconfigured proxy for use through http_proxy= environment var
 After=network.target
 
 [Service]
-ExecStart=/opt/pac4cli/bin/python /opt/pac4cli/main.py -p 3128 --systemd --loglevel warn
+ExecStart=/usr/bin/pac4cli -p 3128 --systemd --loglevel warn
```

Comment:
yes we need to do that. I'll probably write a `configure` script to do that. Didn't make my mind about a solution yet